src/components/Navigation/Nav.svelte

```diff
@@ -51,5 +53,16 @@
 <nav>
   <ul>
     <li><a aria-current='{segment === undefined ? "page" : undefined}' href='.'>home</a></li>
+    <li>
+      <svelte:component 
+        this={Dropdown}
+        urlBase={"/room/"}
+        dropDownTitle={"rooms"} 
+        dropDownOptions={["create", "subscribed"]}
+      >   
+      </svelte:component>
+    </li>
+    <li><a aria-current='{segment === undefined ? "page" : undefined}' href='.'>friends</a></li>
+    <li><a aria-current='{segment === undefined ? "page" : undefined}' href='.'>enemies</a></li>
```

Comment:
These routes don't _really_ make sense, unless we figure out an XSS-resistant way to have user authentication. Otherwise, any list of friends and enemies is pretty much "publicly available".